deployment/frontend/src/components/search/MetadataModifiedFacet.tsx

```diff
@@ -0,0 +1,134 @@
+import { Filter } from '@/interfaces/search.interface';
+import { Disclosure, Transition } from '@headlessui/react';
+import { ChevronDownIcon } from '@heroicons/react/20/solid';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { Input } from '../_shared/SimpleInput';
+
+export default function MetadataModifiedFacet({
+    setFilters,
+    filters,
+}: {
+    setFilters: Dispatch<SetStateAction<Filter[]>>;
+    filters: Filter[];
+}) {
+    const getUpdatedState = () => {
+        const sinceValue = filters.find(f => f?.key === 'metadata_modified_since')?.value ?? '';
+        const beforeValue = filters.find(f => f?.key === 'metadata_modified_before')?.value ?? '';
+
+        return { metadata_modified_since: sinceValue, metadata_modified_before: beforeValue };
+    };
+
+    const [optionsState, setOptionsState] = useState<{
+        metadata_modified_since?: string;
+        metadata_modified_before?: string;
+    }>(getUpdatedState());
+
+    useEffect(() => {
+        setOptionsState(getUpdatedState());
+    }, [filters]);
+
+    const handleDateChange = (key: string, value: string) => {
+        setOptionsState(prev => ({ ...prev, [key]: value }));
+
+        const newSince = key === 'metadata_modified_since' ? value : optionsState.metadata_modified_since;
+        const newBefore = key === 'metadata_modified_before' ? value : optionsState.metadata_modified_before;
+
+        if (newSince && newBefore && new Date(newSince) > new Date(newBefore)) {
+            alert('Invalid date range: "Since" date must be before the "Before" date.');
+            setOptionsState(prev => ({ ...prev, [key]: '' }));
+            return;
+        }
+  
+        if (newSince && new Date(newSince) > new Date()) {
+            alert('Invalid date range: "Since" date must be today or in the past.');
```

Comment:
Do not use alert, instead, use notify https://github.com/wri/wri-odp/blob/dev/deployment/frontend/src/utils/notify.ts
You can use the "error" severity

Reply:
Ah, okay, I didn't know about that one. I'll update this.

Reply:
Updated